SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialEventTracer.cpp

```diff
@@ -244,19 +168,34 @@ void SpatialEventTracer::BeginOpsForFrame()
 
 void SpatialEventTracer::AddEntity(const Worker_AddEntityOp& Op, const FSpatialGDKSpanId& SpanId)
 {
-	TraceEvent(FSpatialTraceEventBuilder::CreateReceiveCreateEntity(Op.entity_id), /* Causes */ SpanId.GetConstId(), /* NumCauses */ 1);
+	Worker_EntityId EntityId = Op.entity_id;
+	TraceEvent(FSpatialTraceEventName::ReceiveCreateEntityEventName, "", SpanId.GetConstId(), 1,
```

Comment:
If these are single-use strings I think we can just inline them, the compiler/linker optimizes the storage/duplicate cost (usually by eliminating all but 1 identical strings)